arwen/contexts/metering_test.go

```diff
@@ -279,80 +282,212 @@ func TestMeteringContext_GasUsed_NoStacking(t *testing.T) {
 	require.Equal(t, uint64(1401), gasUsedByContract)
 }
 
-func TestMeteringContext_GasUsed_StackOneLevel(t *testing.T) {
-	t.Skip()
+func setUpStackOneLevel(t *testing.T) (*meteringContext, *outputContext, *contextmock.RuntimeContextMock, *vmcommon.ContractCallInput, uint64, *vmcommon.ContractCallInput) {
```

Comment:
Suggest splitting this setup function into smaller ones. It returns far too many things.

For example, returning the mocked `host` already provides access to its inner contexts, so you won't have to return them individually.